packages/database/src/changeHandlers/ChangeHandler.js

```diff
@@ -35,17 +25,16 @@ export class ChangeHandler {
    */
   debounceTime = 1000; // ms
 
-  /**
-   * @type {Record<string, Array>} Changes by record type
-   */
-  changeQueue = {};
+  changeQueue = [];
```

Comment:
Thanks to the tests, I realised that while I was using a single promise for changes of all types, I was handling only one type of change before fulfilling that promise. I decided to allow
* separate translators per record type
* a single handler for changes of all types

The idea is that records in multiple tables can trigger the same type of change. This is pattern is already used in `AnalyticsRefresher` and `EntityHierarchyCacher`. Future classes could still look on the translated details to define a custom behaviour based on each change's record type, if need be.